### PR TITLE
Adds Microplastics & changes Plastic Recipe Slightly

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

Adds Microplastics, a reagent acquired by grinding plastic sheets
it's a very, painfully, slow to metabolize reagent, that causes extremely weak organ damage.

changes the recipe for plastic to make use of the Plastic Polymers chemical exclusively, and instead doing the old recipe gives you 10u of Plastic Polymers. To make plastic you now cool that down to 200K or lower to solidify it

you can turn microplastics into plastic polymers by heating them to 600K

**THIS IS NOT A MEME PR, I AM SERIOUS.**

# Wiki Documentation

Would be required to add it to the Guide to Chemistry
Exact damage values are absurdly low, taking between 12 and 15 minutes to be fatal

Also requires updates to the Guide To Chemistry's plastic recipe, to denote it makes plastic polymers instead of sheets, and addition of the Plastic Polymer chem, noting that cooling it down produces plastic sheets.

# Changelog

:cl:  
rscadd: Added Microplastics
tweak: Plastic Polymers are now a used chem
tweak: adjusts the production method of plastic to need plastic polymers to be cooled; polymer recipe is the same as the old plastic recipe
/:cl:

### CODE REVIEW

**Comment by Cartlord** [comment]
lame "joke" p.r., people will forget this exists in a month because we've cultivated a playerbase that doesn't care about anything but gimmicks & powergame and this can't be implemented into a good gimmick or used to win a fight

**Comment by VaelophisNyx** [comment]
this is...not a joke? Like....what says Joke to you about this

**Comment by Cartlord** [comment]
The utter uselessness of the chemical, plus I've been seeing way more people posting about microplastics over the last few weeks - I assumed this was just part of that _weird_ trend.

**Comment by VaelophisNyx** [comment]
yeah no its unrelated to that, lol

but it's still useful, it's a raw organ damage chem for a broad sweep of organs
sure it's slow, but it's effective since so many people miss the early organ damage notifications

**Comment by yacabo** [comment]
Why is this good for the game?

**Comment by 00ze-cyclone** [comment]
Make it a toxin since its only purpose is poisoning people.

**Comment by VaelophisNyx** [comment]
> Make it a toxin since its only purpose is poisoning people.

problem was as a toxin it absolutely fucking body-slams your liver and it makes it INFINITELY more deadly than intended.

**Comment by VaelophisNyx** [comment]
> Why is this good for the game?

because funny
also slow organ-only poison good

**Comment by N3D6** [comment]
Wow people love complaining about nothing. This is funny and a good pr

**Comment by CMOisLing** [comment]
> Wow people love complaining about nothing. This is funny and a good pr

Double standard

**Comment by N3D6** [comment]
> > Wow people love complaining about nothing. This is funny and a good pr
> 
> Double standard

Bro mad because I didn’t like his nerf of a balanced donator species

**Comment by CMOisLing** [comment]
> > > Wow people love complaining about nothing. This is funny and a good pr
> > 
> > 
> > Double standard
> 
> nerf

**Comment by CMOisLing** [comment]
> > > Wow people love complaining about nothing. This is funny and a good pr
> > 
> > 
> > Double standard
> 
> Bro mad because I didn’t like his nerf of a balanced donator species

Will felinid players ever recover?

**Comment by Addust** [comment]
i can see great potential in this, like get a syringe gun and a buncha syringes full of good ol' microplastics and boom, 25 minutes in organ transplants are going on stationwide

**Comment by Moggram** [comment]
This can be used for new PR chem recipes. (or drinks) I don't get all the dislike for this PR.

**Comment by SomeguyManperson** [comment]
> This can be used for new PR chem recipes. (or drinks) I don't get all the dislike for this PR.

"can be" has a tendency to mean "added then nobody does anything with it because it's a one-off meme pr"

**Comment by VaelophisNyx** [comment]
in this case it was not meant to be a meme. Microplastics are a real present danger IRL and it'd be reasonable that they should be  in game too. Even if for nothing other than a grinding result for plastic sheets

**Comment by VaelophisNyx** [comment]
and yeah this can be built on in the future

**Comment by Cartlord** [comment]
On further thought, this seems like it's best suited as a food-poisoning chemical, and I like that  - we don't have many things that you could feasibly put in food without getting bwoinked for fucking up people's rounds or outright murderbone, but the killing potential of this is low enough that that's not really a concern.

**Comment by VaelophisNyx** [comment]
if this gets closed for being bloat I'mma just stop making new stuff since it'd clear that despite there being a general desire for new stuff, there isn't actually a want for it.

**Review by JamieD1 — APPROVED** [review]

**Comment by JamieD1** [comment]
Says conflicts.. dont see conflicts..?

**Comment by VaelophisNyx** [comment]
> Says conflicts.. dont see conflicts..?

chemical conflicts with the adrenaline nerf

**Comment by Rdrazga** [comment]
If I just did my math right this will kill you in less then 5 minutes, cause traumas long before that, and be deadly at extremely low unit counts.

**Comment by Rdrazga** [comment]
Granted this depends on how many chem ticks there are a second/minute

**Comment by Rdrazga** [comment]
But for what is essentially an invisible killer that is very easy to make, this seems excessively deadly especially to the brain

**Comment by VaelophisNyx** [comment]
PR mentions time to kill based on actual in-game testing
Your math sounds wrong; possible you forgot the mild organ healing you get for being alive? Trust me tho, this stuff is incredibly non-lethal unless you just ignore it entirely

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -2140,3 +2140,21 @@
 	. = ..()
 	REMOVE_TRAIT(L, TRAIT_REDUCED_DAMAGE_SLOWDOWN, type)
 
+/datum/reagent/microplastics
+	name = "Microplastics"
+	description = "Finely ground plastics, reduced to microscopic scale. Nearly unable to metabolize in a body, and potentially harmful in the long term."
+	color = "#ffffff"
+	metabolization_rate = 0.05
```
Comment (Bibby0110) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	metabolization_rate = 0.05 * REAGENTS_METABOLISM
```
might have to adjust your math

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (VaelophisNyx) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: Doesn't that work out to just...One?

**Comment by Sniblet** [comment]
I believe taste_description = "nothing" tells the player "You can taste nothing" which only really makes sense when they're drinking Nothing

**Comment by VaelophisNyx** [comment]
well the idea was if you ingest it at like, the kitchen with dosed food, you aren't IMMEDIATELY told you just ate a slow organ poison. Also unsure if I could even use Null there

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -2140,3 +2140,21 @@
 	. = ..()
 	REMOVE_TRAIT(L, TRAIT_REDUCED_DAMAGE_SLOWDOWN, type)
 
+/datum/reagent/microplastics
+	name = "Microplastics"
+	description = "Finely ground plastics, reduced to microscopic scale. Nearly unable to metabolize in a body, and potentially harmful in the long term."
+	color = "#ffffff"
+	metabolization_rate = 0.05 * REAGENTS_METABOLISM
+	taste_description = "nothing"
```
Comment (Sniblet) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: taste_mult = 0

Comment (VaelophisNyx) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: hmm, forgot about that

**Review by Bibby0110 — APPROVED** [review]
dumb

**Pull request merged**